### PR TITLE
support new create time and size metadata fields

### PR DESCRIPTION
**Pull request by bcavagnolo**

### CODE REVIEW

**Comment by bcavagnolo** [comment]
(note: tests will not pass until datalake-common is released.)

File: setup.py [hunk 1]
```diff
@@ -67,6 +67,7 @@ def get_version():
               'wheel==0.24.0',
               'moto==0.4.22',
               'flake8==2.5.0',
+              'freezegun==0.3.9',
```
Comment (nsp) [inline setup.py:70]: Ooooh, cool!!

File: tests/test_ingester.py [hunk 2]
```diff
@@ -100,7 +100,16 @@ def comparator(actual, expected):
         for a, e in zip(actual, expected):
             err = abs(time.time() - a['start']/1000.0)
             assert err < 5.0
-            assert sort(a['records']) == sort(e['records'])
+            for ar, er, in zip(sorted(a['records']), sorted(e['records'])):
```
Comment (nsp) [inline tests/test_ingester.py]: probably want to use the `sort` function here.

**Review by nsp — APPROVED** [review]
Looks good, tests are passing for me locally

File: tests/test_ingester.py (hunk 2 above)
Comment (bcavagnolo) [reply tests/test_ingester.py]: Ah yes. Fixed.

File: setup.py (hunk 1 above)
Comment (bcavagnolo) [reply setup.py:70]: Yeah. Unfortunately it doesn't work on my timeout-style tests.

**Comment by bcavagnolo** [comment]
Thanks @nsp this should be good to go now.

**Pull request merged**